ui/src/components/RoleForm/index.jsx

```diff
@@ -255,6 +273,24 @@ export default class RoleForm extends Component {
             </SpeedDial>
           </Fragment>
         )}
+        {dialogOpen && (
+          <DialogAction
+            focusOnPrimary
```

Comment:
For delete actions, let's have focus initially set on the less destructive button just in case the user hits enter by accident. I should probably add this to https://github.com/taskcluster/taskcluster/blob/master/dev-docs/best-practices/ui.md 😅